bentoctl/cli/__init__.py

```diff
@@ -105,9 +105,16 @@ def deploy(name, operator, bento, display_deployment_info, file):
             deployment_config_path = save_deployment_config(
                 deployment_config.deployment_config, Path.cwd()
             )
+            deployment_config_path_relative = deployment_config_path.relative_to(
```

Comment:
Will we always expect a Path instance here?

Reply:
yep, the `save_deployment_config()` above creates and returns the config path as a `Path` instance.